daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -167,6 +167,7 @@ object SValue {
 
     def apply(xs: Iterator[(SValue, SValue)]): SGenMap = {
       type O[_] = TreeMap[SValue, SValue]
+      xs.foreach { case (k, v) => comparable(k) -> v }
```

Comment:
What does the `-> v` do?

Reply:
you can take a look at `scala.Predef` 

```scala
 implicit final class ArrowAssoc[A](private val self: A) extends AnyVal {
    @inline def -> [B](y: B): (A, B) = (self, y)
    ...
  }
```

You problably already saw/used that when building map, for instance:

```scala
Map(1 -> "1", 2 -> "2", 3 -> "3)
```

or update a Map, for instance 

```scala
map + (2 -> "4")
```

Reply:
Good catch. Thanks